core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-answer-group-modal.component.html

```diff
@@ -141,5 +141,16 @@ <h3>Add Response</h3>
     .oppia-correctness-label-editor {
       margin-top: 0;
     }
+    .answer-text-lable {
+      font-size: 1.5rem;
+    }
+    .editing-click-tell-learner {
```

Comment:
Can we just define this rule in the existing `oppia-click-to-start-editing` class?